docs/ReleaseHistory.md

```diff
@@ -11,6 +11,16 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- DEP: `System.Buffers` dependency has been removed. The package no longer has any dependencies.
+- BRK: .NET 6.0 and 7.0 are no longer supported as they have reached end-of-life. Use a [supported version version of .NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core): .NET 8.0 or greater. 
```

Comment:
>.NET 8.0 or greater [](http://example.com/codeflow?start=197&length=19)

This guidance won't age, i.e., at some point .NET 8.0 itself won't be recommended. So I suggest removing this and just leaving the stable link.

If it was important to leave this marker, I might add something like `as of 2/20/2025` or something like that (and the freshness of the information would be inferable from the release date of the package). But honestly, just zap it I think.